### PR TITLE
Add nrfmicro_13_52833 support

### PR DESCRIPTION
**Pull request by pashutk**
This PR adds support for E73-2G4M08S1E (nrf52833-based module from EBYTE)  to use with nrfmicro 1.3+ board instead of E73-2G4M08S1C.

Since E73-2G4M08S1E is cheaper and more or less available on the aliexpress for reasonable price, 52833-based nrfmicro could be seen as affordable alternative to 52840-based nrfmicro. Also for a wireless split builds there is no need to have 1mb flash memory and 512kb is enough in most cases.

IMG_6500

Nrfnicro boards with 2G4M08S1E module has been battle-tested on several builds (fifi, helix, skeletyl) by myself and I'm aware of at least several boards built by @krikun98. I hope he will also provide more details on how 1.11 pin now works.

### CODE REVIEW

**Comment by krikun98** [comment]
833_screen
I've built several boards with this firmware, both with nrfmicros and bare E73-2G4M08S1E modules. This is built for a softdevice-enabled bootloader, but there's still enough memory left for screens (as depicted) and RGB (haven't tested that, but the firmware builds). I've also built nrfmicros for direct-pin boards (Sweep), so I'm reasonably sure every pin works.

**Comment by krikun98** [comment]
The Pro Micro pin D10 has been reassigned to 1.04, a previously unused pin accessible from the back, as 1.11 is not present on the 2G4M08S1E. Joric's current 1.4.x revision includes a jumper, and existing boards may be modified with an unobtrusive bodge wire, although this isn't necessary for many shields.
P_20220117_185922

**Comment by pashutk** [comment]
> The Pro Micro pin D10 has been reassigned to 1.04, a previously unused pin accessible from the back, as 1.11 is not present on the 2G4M08S1E. Joric's current 1.4.x revision includes a jumper, and existing boards may be modified with an unobtrusive bodge wire, although this isn't necessary for many shields.
> 
> P_20220117_185922

Can you post the link to 1.4.X revision here please?

**Comment by krikun98** [comment]
> Can you post the link to 1.4.X revision here please?

It's his main branch now: https://github.com/joric/nrfmicro/

File: app/boards/arm/nrfmicro/arduino_pro_micro_pins_52833.dtsi [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+		gpio-map
+			= <0 0 &gpio0 2 0>	/* D18/A0 */
+			, <1 0 &gpio0 29 0>	/* D19/A1 */
+			, <2 0 &gpio0 31 0>	/* D20/A2 */
+			, <3 0 &gpio0 30 0>	/* D21/A3 */
+			, <6 0 &gpio0 20 0>	/* D4/A6 */
+			, <7 0 &gpio0 24 0>	/* D6/A7 */
+			, <8 0 &gpio0 10 0>	/* D8/A8 */
+			, <9 0 &gpio1 6 0>	/* D9/A9 */
+			, <10 0 &gpio1 11 0>	/* D10/A10 */
```
Comment (megamind4089) [inline app/boards/arm/nrfmicro/arduino_pro_micro_pins_52833.dtsi:50]: Analog pin should also be changed to `<10 0 &gpio1 4 0>`

Comment (krikun98) [inline app/boards/arm/nrfmicro/arduino_pro_micro_pins_52833.dtsi:50]:
```suggestion
			, <10 0 &gpio1 4 0>	/* D10/A10 */
```

Comment (pashutk) [reply app/boards/arm/nrfmicro/arduino_pro_micro_pins_52833.dtsi]: @megamind4089 done cc: @krikun98

Comment (krikun98) [reply app/boards/arm/nrfmicro/arduino_pro_micro_pins_52833.dtsi]: I'm sure I've changed that. Must've been lost in the rebasing.

Comment (krikun98) [reply app/boards/arm/nrfmicro/arduino_pro_micro_pins_52833.dtsi]: Oh, I changed that in the main `pro_micro` block and missed it here. Thanks, @megamind4089!

File: app/boards/arm/nrfmicro/nrfmicro_13_52833.zmk.yml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+file_format: "1"
+id: nrfmicro_13_52833
+name: nRFMicro 1.3/1.4
```
Comment (petejohanson) [inline app/boards/arm/nrfmicro/nrfmicro_13_52833.zmk.yml:3]:
```suggestion
name: nRFMicro 1.3/1.4 (nRF52833)
```

**Review by petejohanson — CHANGES_REQUESTED** [review]
One catch on the metadata file.

File: app/boards/arm/nrfmicro/nrfmicro_13_52833.zmk.yml (hunk 2 above)
Comment (pashutk) [reply app/boards/arm/nrfmicro/nrfmicro_13_52833.zmk.yml:3]: Makes sense, thanks

**Comment by petejohanson** [comment]
Can you please rebase on latest `main` and force push? Docs deploy failing, probably from missing some fixes there. Thanks!

**Comment by pashutk** [comment]
> Can you please rebase on latest `main` and force push? Docs deploy failing, probably from missing some fixes there. Thanks!

Screwed up commit's dates at first but now the history seems fine and it builds

**Review by petejohanson — APPROVED** [review]
Thanks, looks good.

**Pull request merged**